mobile/lib/data/datasources/remote/campaign.datasource.dart

```diff
@@ -81,7 +79,11 @@ class CampaignDataSource {
     return CampaignModel.fromJson(response.body);
   }
 
-  Future<void> joinCampaign(int campaignId) async {}
+  Future<void> joinCampaign(int campaignId) async {
```

Comment:
Because It returns `Future`, if not handle it, dont need to `async` `await` here
``` dart
Future<void> joinCampaign(int campaignId) => _dioHelper.post( '${Endpoints.campaigns}/$campaignId/volunteers');
```